cmd/resource/status/cmd/print.go

```diff
@@ -19,11 +19,12 @@ const (
 	statusColumn    = "status"
 	messageColumn   = "message"
 
-	ESC          = 27
-	RED    color = 31
-	GREEN  color = 32
-	YELLOW color = 33
-	WHITE  color = 37
+	RESET         = 0
+	ESC           = 27
+	RED     color = 31
+	GREEN   color = 32
+	YELLOW  color = 33
+	DEFAULT color = -1
```

Comment:
Is this a valid code?

https://en.wikipedia.org/wiki/ANSI_escape_code

Reply:
No, it is not. It is used as a way to signal that we don't want to set any color, just use whatever is default. I will add a comment on this.
I tried using a separate parameter to signal that we didn't want to use a color, but this felt like a slightly better solution.